### PR TITLE
Add optional logging container, run compose in background

### PR DESCRIPTION
**Pull request by Fishbowler**
* Add a dozzle container to view container level logs together in a lightweight UI
* Alter running to go to the background rather than run in the console

### CODE REVIEW

**Comment by guusdk** [comment]
I'm unsure what Dozzle exactly exposes. From a quick run, it seems to offer a webpage that you can use to review the std-out logging. Was this what you were after (notably: it doesn't expose openfire.log, I think)?

Maybe add a line in the readme to explain new users that this is available?

And something that reveals my inexperience: now that the docker startup script daemonizes ... how do I stop things, without having to stop each individual container using `docker stop <name>` ?

**Comment by Fishbowler** [comment]
>  it doesn't expose openfire.log

This is step 1, that's step 2. Definitely part of the plan.

>  how do I stop things, without having to stop each individual container using `docker stop <name>`

Very good point. Let me fix that. Would `./start.sh -s` or a `./stop.sh` do? This combats the current problem that might be unique to my setup - I get sick of the `start.sh` stealing my console.

**Comment by guusdk** [comment]
Let's not be Microsoft and use 'start' to stop something. :)

File: start.sh [hunk 1]
```diff
@@ -53,7 +58,10 @@ case $CLUSTER_MODE in
            COMPOSE_FILE_COMMAND+=("-f" "docker-compose-federated.yml");;
 esac
 
-"${COMPOSE_FILE_COMMAND[@]}" down
```
Comment (Fishbowler) [inline start.sh:56]: Is this the right thing to do, given we've got a separate stop script?
Successive runs of the start will replace containers only if they've been changed. In two minds.
Pro: Nicely separates start from stop
Con: Replacing the stack is 2 steps, not 1

Comment (guusdk) [reply start.sh:56]: I'm not sure if I understand what you're saying. Starting things twice (without a 'stop' in between) is guaranteed to be problematic, becaue of port collisions. An implicit "first bring the old system down" which I think this is, might be sensible. In either case, not a hill for me to die on. We can test how one approach works, and change it later if we find we prefer another approach.

Comment (Fishbowler) [reply start.sh:56]: Starting twice won't be a problem. Docker takes of that, and updates the existing stack. The "down" does more than stop the containers - it'll remove data too. I think you're right though - a clean start feels better. I'll revert that line.

**Pull request merged**